### DISCUSSION TITLE
I find the folder android with three folders of obb media and data in my sd card. What should I do

### DISCUSSION THREAD

**Discussion by adnanmuf2024** [d4rken-org/sdmaid-se#1175] [Q&A]
**Describe the bug**
I found a folder in my sd card titled Android and it has three folders obb media and data just like the system folder on the main storage on phone. What should I do, how did this happened. Even new apps folders are appearing under data of sd card android folder. 
What should I do. 

**To Reproduce**

**Expected behavior**

**Smartphone (please complete the following information):**
 - Device: [cricket wireless 5g dream ]
 - OS: [e.g. Android 11] 
 - Root status [not rooted but used sd maid pro with shizuku 2]
 - SD Maid 2/SE version [e.g. v0.6.6-rc6]

**Debug log**
If you can reproduce the bug, record the event in [a debug log](https://github.com/d4rken-org/sdmaid-se/wiki/Bugs#debug-log).

**Screenshots/Video**
If applicable, add screenshots or a video if it helps to explain your problem.

**Additional context**

**Comment by d4rken** [comment]
Do nothing, this is expected. If you switch the sdcard between multiple devices, consider running the corpsefinder to remove folders belonging to apps that are not installed.
